### PR TITLE
Glslang spv rejects SPIR-V compiles for old versions of GLSL code

### PR DESCRIPTION
**Pull request by dneto0**
Adjust tests accordingly.

### CODE REVIEW

**Comment by Qining** [comment]
lgtm. I see all the --target-env stuffs are included.

File: glslc/test/include.py [hunk 1]
```diff
@@ -125,12 +132,14 @@ class VerifyCompileIncludeTwoSiblings(expect.ValidObjectFile):
...
         File('b.vert',
-             """#include "a"
+             """#version 140
+             #include "a"
              void bfun(){afun();}
              #include "c"
              """),
         File('a',
-             """void afun(){}
+             """
```
Comment (dekimir) [inline glslc/test/include.py]: Why was this newline necessary?  Is there a another latent bug in the `#include`d content scanner?

File: glslc/test/include.py [hunk 2]
```diff
@@ -142,21 +151,23 @@ class VerifyNestedIncludeAmongSiblings(expect.StdoutMatch):
...
         File('b', 'content b\n#include "c"\n'),
         File('c', 'content c\n')])
 
     glslc_args = ['-E', 'a.vert']
 
     expected_stdout = \
-"""#extension GL_GOOGLE_include_directive : enable
+"""#version 140
+#extension GL_GOOGLE_include_directive : enable
 #line 0 "a.vert"
```
Comment (dekimir) [inline glslc/test/include.py]: How is this still correct?

**Comment by dekimir** [comment]
Looks very good overall.  Apparently, some of the added newlines aren't strictly necessary; I'd avoid those in `#include` tests, which were crafted to target specific parser bugs.  Also I'm not sure about the relationship between `#version` and the first `#line` in preprocessor output.

File: glslc/test/include.py (hunk 1 above)
Comment (dneto0) [inline glslc/test/include.py]: Turns out it's not necessary.

File: glslc/test/include.py (hunk 2 above)
Comment (dneto0) [inline glslc/test/include.py]: I haven't modified any of the glslang #line processing or #include processing.
Are you objecting to the line number still being 0?  
I suspect that the #extension line is injected as a preamble that doesn't count toward line numbers in a.vert.  Also, #line number handling semantics changed in some version, I can't remember which.

File: glslc/test/include.py (hunk 1 above)
Comment (dneto0) [inline glslc/test/include.py]: Later commit has reverted to the previous text in this case.

File: glslc/test/include.py (hunk 2 above)
Comment (dekimir) [inline glslc/test/include.py]: I was referring to the fact that a line was inserted into `a.vert` so all `#line`s in the output should have shifted, yet not all did.  But I withdraw my comment on the basis that there's no spec for this, and we can declare `#version` special.

**Comment by dekimir** [comment]
+2

**Comment by dneto0** [comment]
Thanks. I'll squash my 2 commits together, then push to master.

**Comment by dneto0** [comment]
Pushed into master as 2 commits ending with fab641131

**Pull request closed without merging**